magicbus/base.py

```diff
@@ -15,6 +15,10 @@
 import random
 try:
     import select
+    try:
+        epoll = select.epoll
+    except AttributeError:
+        epoll = None
 except ImportError:
     select = None
```

Comment:
This is a bit nicer structuraly.
```suggestion
except ImportError:
    select = None
else:
    try:
        epoll = select.epoll
    except AttributeError:
        epoll = None
```